src/Symfony/Component/HttpFoundation/Session/Storage/MockArraySessionStorage.php

```diff
@@ -58,13 +58,11 @@ class MockArraySessionStorage implements SessionStorageInterface
     protected $metadataBag;
 
     /**
-     * @var array
+     * @var array|SessionBagInterface
```

Comment:
I guess there is no perfect solution. It is an `array`, so the first type declaration is true. And it can be treated as list of `SessionBagInterface`, so the other one is also true.

In such cases, I prefer to have both types defined, as both are useful. But I could delete `array` if you consider it unnecessary.